JSDemos/Demos/Toast/Stack/description.md

```diff
@@ -1 +1,21 @@
-Zdes' morzet byt' vasze opisanie
\ No newline at end of file
+The DevExtreme Toast components can stack multiple notifications. Use the [notify(message, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_stack) or [notify(options, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_stack) method to display stacked messages.
+
+These methods use a **stack** object that has the following structure: *{position, direction}*.
+
+## Specify Position
+
+You can set the `position` field to a string (select *'predefined'* in the radio group) or an object (select *'coordinates'* in the radio group). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, the demo disables the *'bottom'* field, and vice versa.
+
+## Specify Direction
+
+The `direction` field specifies two options: which way the notification stack grows and whether new notifications appear at the end or in the beginning of the line. That's why you'll see pairs of values such as *'up-push'* and *'up-stack'*.
```

Comment:
1. ...`and whether new notifications appear at the end or in the beginning of the line.`

Please change "in the beginning" to "at the beginning".

2. `That's why you'll see pairs of values such as *'up-push'* and *'up-stack'*.`

I'd like to make this sentence sound less casual, and specify _what_ shows the *'up-push'* and *'up-stack'* values.
What do you think about the following?

`For this reason, the field's pull-down menu choices show pairs of values such as *'up-push'* and *'up-stack'*.`